crates/ruff_linter/src/rules/pydocstyle/snapshots/ruff_linter__rules__pydocstyle__tests__D208_D.py.snap

```diff
@@ -109,7 +108,7 @@ D.py:675:1: D208 [*] Docstring is over-indented
 673 |         This is overindented
 674 |             And so is this, but it we should preserve the extra space on this line relative
 675 |             to the one before
-    |  D208
```

Comment:
This range seems nonsensical. It should probably highlight the over-indented part but that's an improvement for another day.

Reply:
Yeah I believe this lint just always points to the start of the corresponding line.